things/redis/streams_test.go

```diff
@@ -65,16 +65,20 @@ func TestAddThing(t *testing.T) {
 		event map[string]interface{}
 	}{
 		{
-			desc:  "create thing successfully",
-			thing: things.Thing{Type: "app", Name: "a", Metadata: "metadata"},
-			key:   token,
-			err:   nil,
+			desc: "create thing successfully",
+			thing: things.Thing{
+				Type:     "app",
+				Name:     "a",
+				Metadata: map[string]interface{}{"test": "test"},
+			},
+			key: token,
+			err: nil,
 			event: map[string]interface{}{
 				"id":        "1",
 				"name":      "a",
 				"owner":     email,
 				"type":      "app",
-				"metadata":  "metadata",
+				"metadata":  "{\"test\":\"test\"}",
```

Comment:
Why not `map[string]interface{}{"test": "test"},` ?

Reply:
Because this is sent to Redis Stream, and there you cannot use `map[string]interface{}`.

Reply:
ok